src/test/java/com/wikia/webdriver/testcases/visualeditor/VEVideoTests.java

```diff
@@ -34,21 +34,22 @@ public void addNonPremiumVideo() {
     veNew.verifyVEToolBarPresent();
     veNew.publish();
   }
-
-  @Test(groups = {"VEVideo", "VEAddExternalVideo"})
-  @Execute(asUser = User.USER)
-  public void addPremiumVideo() {
-    VisualEditorPageObject ve =
-        new VisualEditorPageObject().openVEOnArticle(wikiURL, PageContent.ARTICLE_NAME_PREFIX
-                                                                    + DateTime.now().getMillis());
-    VisualEditorPageObject veNew = ve.addVideoToContent(VideoContent.PREMIUM_VIDEO_URL);
-    veNew.verifyVideos(1);
-    veNew.verifyVEToolBarPresent();
-    veNew.publish();
-  }
+//  // I think this test should be removed, because Premium Video was something from video.wikia.com
+//  @Test(groups = {"VEVideo", "VEAddExternalVideo"})
+//  @Execute(asUser = User.USER)
+//  public void addPremiumVideo() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)